extensions/amp-image-viewer/0.1/amp-image-viewer.css

```diff
@@ -0,0 +1,5 @@
+.i-amphtml-image-viewer-image {
+  position: absolute;
```

Comment:
actually, I don't think we should have this. `amp-carousel` should apply layout fill to it. lightbox can also customize the CSS for it if needed.

Reply:
This applies to the image that the `amp-image-viewer` creates. It has to be `position: absolute` because that's how zoom / pan manipulates its positioning. We can apply the styling on the element itself if that is better.

Reply:
Oh I see, my bad missed the `-image` at the end.